src/main/resources/db/changesets/20240419_addSnapshotId.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: addSnapshotId
+      author: rjohanek
+      changes:
+        - addColumn:
+            tableName: snapshot_builder_settings
+            columns:
+              - column:
+                  name: snapshot_id
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: true
+                    unique: true
```

Comment:
So, when I remove the unique constraint I get a bad SQL exception because it says the constraint "fk_snapshot_snapshot_builder_settings" does not exist, but it works as expected when marked true